.github/workflows/build_documentation.yml

```diff
@@ -76,41 +77,24 @@ jobs:
           git config --global user.name "Hugging Face Doc Builder"
           git config --global user.email docs@huggingface.co
           
-          cd doc-build
-          git pull origin main
-          cd ..
-
-          cd notebooks
-          git pull origin master
-          cd ..
-
       - name: Make documentation
         run: |
           cd doc-builder &&
-          doc-builder build transformers ../transformers/docs/source --build_dir ../doc-build --notebook_dir notebooks/transformers_doc --clean --html &&
+          doc-builder build transformers ../transformers/docs/source --build_dir ../build_dir --notebook_dir ../notebooks_dir --clean --html &&
           cd ..
         env:
           NODE_OPTIONS: --max-old-space-size=6656
 
       - name: Push to repositories
         run: |
           cd doc-build &&
-          if [[ `git status --porcelain` ]]; then 
-            git add . &&
-            git stash && git pull && git stash apply &&
-            git commit -m "Updated with commit ${{ github.sha }} \n\nSee: https://github.com/huggingface/transformers/commit/${{ github.sha }}" &&
-            git push origin main
-          else
-            echo "No diff in the documentation."
-          fi &&
+          git pull &&
+          cp -r ../builder_dir/transformers . &&
```

Comment:
```suggestion
          cp -r ../builder_dir/transformers ./transformers &&
```

Reply:
In my tests this copies `transformers` inside `transformers/transformers`

Reply:
updated it to be `transformers/master`